public/styles/colors.css

```diff
@@ -39,7 +39,7 @@
 
     --high-violet: #c558f5;
 
-    --wave-jammer: #a31177;
+    --wave-jammer: #600A6A;
```

Comment:
I think prettier will change it to lowercase, wouldn't it?

Reply:
it didnt, but I will